GomokuClient/packages/gomoku-core/src/pages/JoinGame.tsx

```diff
@@ -72,6 +71,36 @@ const JoinGame = ({ gameHistory }: JoinGameProps) => {
       });
       toaster.show("Rematch request sent");
     },
+    currentPlayer:
+      players.black?.playerId === jwtDecodedInfo?.userId ? "black" : "white",
```

Comment:
Do we need to compare with anonymous generatedID as well, to handle this logic for anonymous case, or it will be for another PR?

Reply:
Pushed fallback